src/Microsoft.PowerShell.Commands.Utility/commands/utility/RemoveAliasCommand.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Management.Automation;
+using System.Management.Automation.Internal;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// The implementation of the "Remove-Alias" cmdlet.
+    /// </summary>
+    ///
+    [Cmdlet(VerbsCommon.Remove, "Alias", DefaultParameterSetName = "Default", HelpUri = "")]
+    public class RemoveAliasCommand : PSCmdlet
+    {
+         #region Parameters
+
+        /// <summary>
+        /// The alias name to remove.
+        /// </summary>        
+        [Parameter(Position = 0, Mandatory = true, ValueFromPipeline = true, ValueFromPipelineByPropertyName = true)]
+        public string Name { get; set; }
```

Comment:
This should be a string[]. All the Remove* cmdlets that have a Name parameter support having a array for names.